doc/tutorials/using_ubuntu_as_sos.rst

```diff
@@ -35,6 +35,46 @@ Ubuntu 16.04.4 LTS was used throughout this document, other versions such as
      sudo service ssh status
      sudo service ssh start
 
+Install build tools and dependencies
+************************************
+
+Install development tools for ARCN development:
+
+* On a Ubuntu development system:
+
+  .. code-block:: none
+
+     sudo apt install gcc \
```

Comment:
This would read better if you didn't put each package on a separate line -- put a few on each line keeping line length < 80 characters.

Reply:
Thanks for your notice and I have modified the layout of codes with length  < 80 characters.